src/client/app/style.css

```
@@ -32,95 +90,177 @@ canvas {
+    padding: 5px 9px;
     text-transform: uppercase;
     width: fit-content;
 }
 
+#connection-status::before {
+    content: '';
+    display: inline-block;
+    width: 5px;
+    height: 5px;
+    border-radius: 50%;
+    background: currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)